packages/jest-circus/runner.d.ts

```diff
@@ -0,0 +1,3 @@
+export {default} from './build/legacy-code-todo-rewrite/jestAdapter';
```

Comment:
it's not `default`, and it will only work if you use `esModuleInterop` on your side.

It should be something like

```ts
import adapter = require('./build/legacy-code-todo-rewrite/jestAdapter');
export = adapter
```

You probably need to create a namespace as well in order to add `state` and `run` to that export. Freehand, but something like this, maybe?

```ts
import adapter = require('./legacy-code-todo-rewrite/jestAdapter');
import * as circusState from './state';
import circusRun from './run';

declare const jestCircusRunner = adapter;

// eslint-disable-next-line no-redeclare
declare namespace jestCircusRunner {
  export const state = circusState;
  export const run = circusRun;
}

export = jestCircusRunner;
```

it _might_ be easier to create a `runner.ts` inside of `src/` and create this new exported thing that you want, and have the `runner.js` in root just re-export that?